DESCRIPTION

```diff
@@ -67,7 +67,7 @@ Suggests: bit64, curl, R.utils, xts, nanotime, zoo, yaml, knitr, rmarkdown
 SystemRequirements: zlib
 Description: Fast aggregation of large data (e.g. 100GB in RAM), fast ordered joins, fast add/modify/delete of columns by group using no copies at all, list columns, friendly and fast character-separated-value read/write. Offers a natural and flexible syntax, for faster development.
 License: MPL-2.0 | file LICENSE
-URL: http://r-datatable.com, https://Rdatatable.gitlab.io/data.table, https://github.com/Rdatatable/data.table
+URL: https://rdatatable.gitlab.io/data.table, https://Rdatatable.gitlab.io/data.table, https://github.com/Rdatatable/data.table
```

Comment:
better to just remove this link? presumably no need to include both `https://rdatatable.gitlab.io/data.table` and `https://Rdatatable.gitlab.io/data.table`